### PR TITLE
Download pkgconfig for protobuf in download mode

### PR DESCRIPTION
**Pull request by ThomsonTan**
**Describe the pull request**

- What does your PR fix?
`vcpkg install protobuf --only-downloads` doesn't download mingw tool pkgconfig because the running is halted in configure process, and download of pkgconfig is expected to happen at the end of install process which is after the halt. The breaks the following run of `vcpkg install protobuf --no-downloads` which complains that pkgconfig is unavailable in local cache. 
This PR fixes the issue by explicitly downloading pkgconfig before configure in download mode.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes.

### CODE REVIEW

**Comment by JackBoosY** [comment]
Does this work? I doubt it.

**Comment by ras0219** [comment]
This does work, however it would be much better to fix this in the general case instead of on a port-by-port basis like this would required.

One possibility would be to search for `vcpkg_fixup_pkgconfig` in the `portfile.cmake` from `scripts/ports.cmake` and pre-acquire it there; this is a bit hacky but it would work for our current cases. This is a bit of a fundamental issue with "downloads-only" approach mixed with the imperative nature of `portfile.cmake`.

**Comment by ThomsonTan** [comment]
Thanks @ras0219 . I thought about fix it in general way, but as the current halt execution in varies places according to running process is enabled or not, it looks hard to fix this fundamental problems ideally, even if pkgconfig could be fixed in general but the same issue could happen in other tools.

I observed this on in protobuf so far, so I submitted this fix with minimum impact to unblock some CI build. Free free to add general fix.

**Review by JackBoosY — CHANGES_REQUESTED** [review]
Please bump the port version. See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports).

File: ports/protobuf/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: protobuf
-Version: 3.13.0
+Version: 3.13.0-1
```
Comment (JackBoosY) [inline ports/protobuf/CONTROL:2]:
```suggestion
Version: 3.13.0
Port-Version: 1
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**